sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -980,13 +980,20 @@ static void _l2filter_usage(const char *cmd)
 }
 #endif
 
+static void _cca_usage(const char *cmd)
+{
+    printf("usage: %s <if_id> cca\n", cmd);
+    puts("       perform a clear channel assessment and print the result\n");
```

Comment:
The line feed at the end is IMO unnecessary. When calling `ifconfig help` it produces an empty line between the `cca` and `stat reset` usages. I have no strong opinion about this, but for my taste I would prefer no line feed at all, since the usage text is already indented.